.github/workflows/tests.jsonnet

```diff
@@ -578,8 +605,9 @@ local ignore_md = {
   jobs: {
     'test-semgrep-core': test_semgrep_core_job,
     'test-osemgrep': test_osemgrep_job,
-    // Pysemgrep tests, requires build-test-core-x86 job
+    // Pysemgrep tests that require check-semgrep-pro
```

Comment:
Is any test using the Pro binary except for the new one you added here `test_taint_intrafile` ?

Reply:
No other test here is using the Pro binary.

Just future tests in this PR: https://github.com/semgrep/semgrep/pull/9872